packages/web/src/common/store/tipping/selectors.ts

```diff
@@ -173,3 +165,122 @@ export const getOptimisticSupporters = createSelector(
     return result
   }
 )
+
+export const getOptimisticSupportingForUser = (
+  state: CommonState,
+  userId: ID
+) => getOptimisticSupporting(state)[userId]
+
+export const getOptimisticSupportersForUser = (
+  state: CommonState,
+  userId: ID
+) => getOptimisticSupporters(state)[userId]
+
+export const getOptimisticUserIdsIfNeeded = (
```

Comment:
Was pondering this some more – I don't love that our selector knows about userlist, but we're already in a bit of an awkward state since userlist has to know about the particular list it's showing, and this is just replicating the behavior on mobile. I think this might actually belong in the userlist selectors rather than the tipping selectors, since it also handles the non-tipping tag state of just returning users specifically for the userlist?

Reply:
yeah i buy it. moved!